splunkapp/generate_wazuh_splunk_app.sh

```diff
@@ -88,6 +90,10 @@ clean(){
     exit ${exit_code}
 }
 
+ctrl_c() {
+    clean 0
```

Comment:
Killing the process with a SIGKILL must return anything distinct from 0.

Reply:
Done in https://github.com/wazuh/wazuh-packages/pull/329/commits/2f591e77f6f402eed58bb450c62a213ff1d5f8cc.